solution/backend/regulations/templates/regulations/resources.html

```diff
@@ -0,0 +1,24 @@
+{% extends "regulations/base.html" %}
+
+{% load static %}
+
+{% block title %}Cache Explorer{% endblock %}
```

Comment:
Is this the right title?

Reply:
Negative.  Let me change it.

Reply:
This route gets a lot of attention in the story I'm currently working on as well